import-dashboards.py

```diff
@@ -164,6 +164,27 @@ def add_datasources(api_key):
     r = requests.get('%s/api/datasources' % (HOST,), headers=grafana_headers(api_key))
     print ' * Datasources: %r %r' % (r.status_code, r.content)
     ds = [x['name'] for x in json.loads(r.content)]
+    if 'Metrics' not in ds:   # https://jira.percona.com/browse/PMM-6518
```

Comment:
This function (`add_datasources`) for me looks too long.
Maybe it is time to split it?
```python
def add_datasources(api_key):
    _add_meterics_datasources(api_key)
    _add_prometheus_datasources(api_key)
    # ....
    _add_alert_manager_datasources(api_key)
```

Reply:
It has been splitted,